### PR TITLE
Improves Rectangle tool performance.

### PR DESCRIPTION
**Pull request by smiegrin**
This commit improves the performance of the rectangle drawing tool, especially for larger pen sizes.

---
This change is [Reviewable](https://reviewable.io/reviews/juliandescottes/piskel/618)

### CODE REVIEW

File: src/js/tools/drawing/Rectangle.js [hunk 1]
```diff
@@ -20,10 +20,19 @@
    * @override
    */
   ns.Rectangle.prototype.draw = function (col, row, color, targetFrame, penSize) {
-    var rectanglePixels = pskl.PixelUtils.getBoundRectanglePixels(this.startCol, this.startRow, col, row);
+    var rectangle = pskl.PixelUtils.getOrderedRectangleCoordinates(this.startCol, this.startRow, col, row);
```
Comment (juliandescottes) [inline src/js/tools/drawing/Rectangle.js]: Can you just remove getBoundRectanglePixels from PixelUtils since no other file is using it now?

**Comment by juliandescottes** [comment]
Great @smiegrin thanks for fixing this! 
Can you cleanup PixelUtils? I'll merge right after!

**Comment by juliandescottes** [comment]
I'm merging, I'll just push a follow up commit

**Comment by smiegrin** [comment]
Sorry, I made the fix, but it seems that I didn't push it to the right branch. I'll try to fix that today.

**Comment by juliandescottes** [comment]
No worries! I pushed the follow up already though.

Thanks for the fix again!

**Pull request merged**